.github/workflows/cmake.yml

```
@@ -47,3 +47,13 @@ jobs:
     - name: Test
       working-directory: ${{github.workspace}}/build
       run: ctest --output-on-failure
+
+    - name: Install
+      run: cmake --install ${{github.workspace}}/build --strip --prefix ${{github.workspace}}/install
+
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        name: unshield-${{matrix.os}}-${{github.sha}}
+        path:  ${{github.workspace}}/build
```

Does it make sense to use the installed files for the upload?
```suggestion
        path:  ${{github.workspace}}/install
```

---

Good catch.